cmp/grid/columns/Column.ts

```diff
@@ -184,6 +184,11 @@ export interface ColumnSpec {
      */
     absSort?: boolean;
 
+    /**
+     * True to always sort null / undefined values to the bottom, regardless of sort order.
+     */
+    bottomNullSort?: boolean;
+
     /**
```

Comment:
Would we want to extend this to NaN, '',  undefined?  We should consider which other sentinel values (other than zero) might warrant the same treatment?  We could even have a *list* of values, that should be put at the bottom, with the order they should appear there.  Would this be useful for values like 'CLOSED'?  Or other high-level ordering we want to impose independent of the sorting.

Reply:
could be something like `sortToTop:[]` `sortToBottom:[]`

Reply:
I really like this idea - makes it more flexible and generalizable. We could support an array of values of equality checks, or boolean closures for more complex checks (e.g. `v => isNil(v) || itEmpty(v)`)

Reply:
For values like 'CLOSED', user expectations have been that the 'CLOSED' record be at the bottom no matter which column is sorted, and no matter which column has the 'CLOSED'.  So, not sure how useful this would be, unless the comparators allow getting the value from other fields.

Reply:
Yeah, I don't think we can use this for cross-column sorting like you describe. I think that's kind of an edge case and not necessarily something we should support with hoist out of the box, where the 'normal' use case for sorting is only concerned with the sorted column.

Reply:
tom -- simplified slightly, tweaked performance, and also allowed it to take a single value.

Let me know if it makes sense?